.github/workflows/publishWorkflow.yml

```diff
@@ -21,4 +29,4 @@ jobs:
         with:
           access: public
           registry: https://registry.npmjs.org/
-          token: ${{ secrets.NPM_USER_ACCESS_TOKEN }}
+          token: ${{ env.NPM_USER_ACCESS_TOKEN }}
```

Comment:
I am not 100% sure, but given this is a standard env variable, won't it be visible in the public build? 🤔 (and if so, I think it'd allow anyone checking the build action to use said token to publish instead of us 😱 )

Reply:
hmm that's a good point, this was infra's recommendation to make this change. let me ask them.

Reply:
Yeah I assume on private repositories it's ok and shouldn't matter, but on public ones it sounds dangerous 😅

Reply:
They knew this was a public repo, so I'm not sure why they recommended it. I still have to ask them. Leaving them a message now.

Reply:
Link to conversation: https://babbel.slack.com/archives/CR9AC701E/p1732595963440359

Reply:
Thanks for checking, looks like they are confident this is alright so we should be good to go 🚀

Reply:
@jduthon infra confirmed secrets remain masked when using the `aws-actions/aws-secretsmanager-get-secrets` action (which we are). So we're good to go here.